src/features/list-view/__tests__/MetadataListViewMessage-test.js

```diff
@@ -1,7 +1,7 @@
 // @flow
 import * as React from 'react';
 
-import Message from '../Message';
+import Message from '../components/MetadataListViewMessage';
 
```

Comment:
The test at the bottom of this file uses a native Array forEach, if you would prefer I can refactor into a test.each

Reply:
yeah would be nice to be consistent with that, so test.each